### PR TITLE
Rolling release changelog

### PR DESCRIPTION
**Pull request by Profpatsch**
Updates to the rolling-release branch.

### CODE REVIEW

File: release.nix [hunk 1]
```diff
@@ -5,6 +5,37 @@
   changelog = {
     # Find the current version number with `git log --pretty=%h | wc -l`
     entries = [
+      {
+        version = 309;
+        changes = ''
+          - The deprecated `lorri shell` command was removed.
+          - Fix problem with non-UTF8 nix output, arbitrary bytes are fine.
```
Comment (grahamc) [inline release.nix]:
```suggestion
          - Fix problem with non-UTF8 build output, arbitrary bytes are now fine.
```

File: release.nix [hunk 2]
```diff
@@ -5,6 +5,37 @@
...
+            - Take `PWD`, `OLDPWD` and `SHELL` from user environment
+              instead of shell file context.
+            - Set `IN_NIX_SHELL` to `impure`.
+              - Fixes SSL certificates being set to bogus path.
+
+          - `watch`:
+            - Add `--once` option to exit after one build.
+
+          - Watcher logic:
+            - Emulate nix’s `default.nix` behaviour instead of watching
```
Comment (grahamc) [inline release.nix]:
```suggestion
            - Emulate Nix’s `default.nix` behaviour instead of watching
```

File: CHANGELOG [hunk 3]
```diff
@@ -0,0 +1 @@
+release.nix
```
Comment (grahamc) [inline CHANGELOG]:
```suggestion
See ./release.nix
```

Comment (Profpatsch) [reply CHANGELOG]: Ah, this is a symlink

**Pull request merged**